coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetCommonController.java

```diff
@@ -80,9 +83,9 @@ public ModelAndView start(@RequestParam("budgetId") Long budgetId, @RequestParam
         ModelAndView result = getModelAndViewService().getModelAndViewWithInit(form, ProposalBudgetConstants.KradConstants.BUDGET_DEFAULT_VIEW);
         form.setViewOnly(inViewMode);
         form.setCanEditView(!inViewMode);
-        if (budgetRatesService.checkActivityTypeChange(form.getBudget().getBudgetRates(), form.getDevelopmentProposal().getActivityTypeCode())) {
+        if (!inViewMode && budgetRatesService.checkActivityTypeChange(form.getBudget().getBudgetRates(), form.getDevelopmentProposal().getActivityTypeCode())) {
```

Comment:
@blackcathacker  good catch. 
Initially I tried to set it up with view.readOnly which should work for all possible options.
I saw some issues/conflict with budget settings and to set budget in view mode when PD is opened in view mode.
Looking into it to see if that will still work as a global solution.